src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.ts

```diff
@@ -174,9 +254,9 @@ export class StudentExamDetailComponent implements OnInit {
     }
 
     getWorkingTimeToolTip(): string {
-        return this.examIsVisible()
-            ? 'You cannot change the individual working time after the exam has become visible.'
-            : 'You can change the individual working time of the student here.';
+        return this.canChangeExamWorkingTime()
```

Comment:
Should we use a regular ``if`` here for increased readbility?

Reply:
:+1: Done in 9db1dc851017457efe4d6d88648e9ac377202032.